package.json

```diff
@@ -4,22 +4,15 @@
   "private": true,
   "license": "AGPL-3.0",
   "scripts": {
-    "build": "yarn build-shared",
-    "cleanbuild": "yarn clean-all && yarn build-all",
-    "build-all": "CYPRESS_INSTALL_BINARY=0 yarn build-shared && yarn build-legacy && yarn build-ui && yarn build-root && yarn copy-build",
-    "build-root-css": "cd root && [ -f dist/assets/css/root-application.css ] || { yarn install; sass --load-path=../node_modules src/scss/base.scss dist/assets/css/root-application.css; }",
+    "build": "CYPRESS_INSTALL_BINARY=0 yarn --cwd ui install && yarn --cwd ui build && yarn copy-build",
     "build-shared": "cd shared && yarn install && yarn build",
-    "build-legacy": "cd legacy && yarn install && yarn build",
-    "build-root": "cd root && yarn install && yarn build",
-    "build-ui": "cd ui && yarn install && yarn build",
+    "build-all": "yarn build",
```

Comment:
This script is run by the backend when it builds the UI so we need to maintain the name for now.